docs/source/concept_models/vivarium_swissre_colorectalcancer/concept_model.rst

```diff
@@ -479,49 +506,43 @@ Relevant formulas
      - Source
      - Note
    * - MST
-     - 4.5-5.8 years
+     - 4.5-5.8 years depending on age
      - [Brenner-et-al-2012]_
-     - 
+     - Let's use a MST of 5 years for all ages.
    * - Colon and rectum cancer screening coverage in baseline
-     - 20%
+     - 15%
      - [lin-et-al-2019]_
      - 
    * - Colon and rectum cancer screening covearge in alternative scenario
      - scale up to 60% in 2030
      - [de-Moor-et-al-2018]_
-     - We applied 20-year lag on US coverage
+     - We applied a 20-year lag on US coverage.
    * - FOBT sensitivity
      - 68% (95%CI 57%-79%)
      - [Meklin-et-al-2020]_
-     - We documented the sensitivity of gFOBT
+     - Normal distribution with mean=0.68 and std_dev=0.056
    * - FOBT specificity
      - 88% (95%CI 84%-91%)
      - [Meklin-et-al-2020]_
-     - We documented the specificity of gFOBT
+     - Normal distribution with mean=0.88 and std_dev=0.018
    * - Colonoscopy sensitivity
      - 98% (95%CI 91%-100%)
      - [Rockey-et-al-2005]_
-     - 
+     - Triangular distribution with left=0.90, mode=0.98, and right=1.00
```

Comment:
Would it be acceptable to use `TruncnormDist('colonoscopy_sensitivity', 0.98, 0.04)` instead?

Reply:
NormalDist(mean=0.98, std_dev=0.04) tends to have a lower of 0.90 and a upper of 1.06. I think it works if we constrain the upper limit to be 1.0.